cpp/include/experimental/graph_generator.hpp

```diff
@@ -47,7 +47,7 @@ namespace experimental {
  * handles to various CUDA libraries) to run graph algorithms.
  * @param scale Scale factor to set the number of verties in the graph. Vertex IDs have values in
  * [0, V), where V = 1 << @p scale.
- * @param num_edges Number of edges to generate.
+ * @param edge_factor Average number of edges per vertex to generate.
```

Comment:
Thank you for finding my mistake!

Actually, this should be num_edges.

https://github.com/rapidsai/cugraph/blob/branch-0.19/cpp/src/experimental/generate_rmat_edgelist.cu#L39

It is num_edges in the code, and the input parameter below ```size_t edge_factor``` should be ```size_t num_edges``` instead of fixing the comment here. This gives more flexibility in generating edges in multiple GPUs.

Reply:
Yea I figured actually I fixed it while you were reviewing haha
I got confused by the name of the parameter in the function, which I fixed